web/src/stream_edit.js

```diff
@@ -603,8 +603,24 @@ export function initialize() {
         const stream = sub_store.get(stream_id);
 
         const stream_name_with_privacy_symbol_html = render_inline_decorated_stream_name({stream});
+
+        const is_new_stream_notification_stream =
+            stream_id === page_params.realm_notifications_stream_id;
+        const is_signup_notification_stream =
+            stream_id === page_params.realm_signup_notifications_stream_id;
+        const is_report_notification_stream =
+            stream_id === page_params.realm_report_message_stream_id;
```

Comment:
There is no such setting currently. I am not sure whether we should include this now or wait till the setting is added since we might possible name the setting something else.

Reply:
As per the discussion [here](https://chat.zulip.org/#narrow/stream/101-design/topic/warning.20when.20archiving.20a.20stream.20used.20for.20announcements.2E/near/1603027), I have removed the code. You can take a look now.